### PR TITLE
Fixed 2 bugs for FAB

### PR DESCRIPTION
**Pull request by first087**
* Fixed Rectangle and Shadow bug
* Fixed Margin bug

### CODE REVIEW

**Comment by jaredsburrows** [comment]
@first087  If you added the `dimen`: `<dimen name="fab_margin">0dp</dimen>`, then use the `fab_margin` instead of using `0dp`.

**Comment by jaredsburrows** [comment]
@first087  Also, the `fab_margin` needs to be `16dp`. Please see https://github.com/chrisbanes/cheesesquare/pull/10.

**Comment by first087** [comment]
@jaredsburrows Plz see this commit --> https://github.com/first087/cheesesquare/commit/6a6d65243a3f74b828af5306975394bf8756c9b7

**Comment by jaredsburrows** [comment]
@first087 Yeah, good job. This is ready for merging.

File: app/src/main/res/layout/activity_detail.xml [hunk 1]
```diff
@@ -157,6 +157,7 @@
         android:layout_width="wrap_content"
         app:layout_anchor="@id/appbar"
         app:layout_anchorGravity="bottom|right|end"
+        app:borderWidth="0dp"
```
Comment (samirshrestha) [inline app/src/main/res/layout/activity_detail.xml]: @first087 default border width is 0dp so it is not required

Comment (first087) [inline app/src/main/res/layout/activity_detail.xml]: Commit message at https://github.com/first087/cheesesquare/commit/99290c69356562976ef5a4bb594aca1003fb9214

> Add `app:borderWidth="0dp"` for FAB Bug
> 
> - Fixed : **Square bug** for Pre-Lollipop
> - Fixed : **Shadow bug** for Lollipop+

**Pull request closed without merging**